### ISSUE TITLE
Random (play/show) 

### ISSUE THREAD

**Issue by Peceha** [diffractor/diffractor#21]
Hi,
I am aware there is a SHUFFLE option to randomize the order of the files but it ends up with a 'playlist' that is still kind of sorted (just randomly, hehe) and which you Can only browse in that new 'shuffled' order.
What about some kind of 'play/show random' shortcut that Can play random file from a playlist? 

Thanks in advance :)

**Comment by ZacWalk** [comment]
The reason for shuffle (and not play randomly) is because of the previous/next item UI. Users would pause a slide show and then want to go back to the previous item. The 'previous item' command shows the previous item as listed in the items view. 

I mainly want to avoid users saying 'where was that amazing photo I just saw 2 photos back. 'Previous item' showed me something else' :) 

@Peceha do you have any thoughts/examples of how play-random could be implemented well. I mean in terms of an intuitive UI 
around next/previous item navigation. Maybe I should just do it the way Spotify does with a shuffle mode button.

**Comment by Peceha** [comment]
Hi,
Now that I know the reason behind it I'm afraid I cannot think of anything usefull.. 

Maybe just a shortcut to choose a random file, select it on the viewlist and play/show (not a continous "random playback/slideshow Mode" - for this the shuffled view list is enough - what I'm taking here about is just one random "jump" )

Let's say there is already a random file selected and it is being played.
All the futher "arrow navigation" (next/prev item) should be counted from that random file position (and so on). 

In other words the user should be aware that pressing PLAY/SHOW RANDOM will make him/her unable to came back to what whas shown before. 

That is how I see it :) 

Thanks..

**Comment by ZacWalk** [comment]
That sounds reasonable to me. My thinking is that I should replace shuffle sorting with 'shuffle play mode'. Probably a few months away though because I have quite a few bugs to fix first.

**Comment by ZacWalk** [comment]
Closed in the latest version (v1.26.3). Please re-open if you still see any issues. Thanks!
